
---
string    version
string[]  plugins_loaded
string    robot_name
